examples/guides/data-driven-viz-2/step-2.html

```diff
@@ -122,7 +118,7 @@ <h2>ramp(globalEqIntervals($total_damage, 3), [1, 25])</h2>
 
                 if (color) {
                     colorLegendList +=
-                        `<li><span class="point-mark" style="background-color:${color}; border: 1px solid black;"></span><span>${legend.key.replace('CARTO_VL_OTHERS', 'Rest of types')}</span></li>\n`;
+                        `<li><span class="point-mark" style="background-color:${color}; border: 1px solid black;"></span><span>${legend.key.replace('CARTO_VL_OTHERS', 'All other causes')}</span></li>\n`;
```

Comment:
The style is by accident type, not the accident cause ( see referenced column)

Reply:
ok, feel free to change "Rest of Types" didn't make sense to me since there wasn't a description of what the map was showing